### DISCUSSION TITLE
Difference in functionality between API and GUI

### DISCUSSION THREAD

**Discussion by Kerby92** [danielhrisca/asammdf#522] [Q&A] [answered]
Hey there,

great work done so far! Really inspiring.

I have one general question. There is an API and GUI interface, but has the GUI full access to all API related functions or is the GUI functionality limited in some way?

This is just for clarification. From my understanding, the API is meant for automation principles.

Thank you!

**Comment by danielhrisca** [comment, marked as answer]
Hey, the GUI should be mostly the same as the API.do you have an exact point were it does not match the api?

**Reply by Kerby92** [reply]
No, I don't have any exact point.

As I said, I only wanted to clarify if my presumption is correct. Thanks for confirming it! :)
